framework/db/mssql/QueryBuilder.php

```diff
@@ -25,6 +25,7 @@ class QueryBuilder extends \yii\db\QueryBuilder
     public $typeMap = [
         Schema::TYPE_PK => 'int IDENTITY PRIMARY KEY',
         Schema::TYPE_BIGPK => 'bigint IDENTITY PRIMARY KEY',
+        Schema::TYPE_CHAR => 'char(1)',
```

Comment:
I thing this could be wrong. `char` in MSSQL stands for non-unicode only same as `varchar`. There are unicode aware `nchar` and `nvarchar`.